dockerfiles/jaeger/agent/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile tracks jaegertracing/jaeger-agent, but includes more debugging tools and runs as a
+# non-root user. It requires JAEGER_VERSION to be set as an argument to build.
+ARG JAEGER_VERSION
+FROM jaegertracing/jaeger-agent:$JAEGER_VERSION as base
+
+FROM alpine:3.11.5
```

Comment:
why do you create a new image? Why not just run commands in the jaeger-agent image?

Reply:
The jaeger-agent image ([defined here](https://github.com/jaegertracing/jaeger/blob/master/cmd/agent/Dockerfile)) uses `scratch` as its base image. Consequently, it contains nothing except the agent binary itself. It doesn't have `apk` or even a shell. If we want to install debugging tools, it's much easier to start from a linux distro base image.